app/src/main/res/values/strings.xml

```diff
@@ -1087,4 +1087,5 @@ Do you want to switch to this account?</string>
   <string name="bitwarden_can_notify_you_each_time_you_receive_a_new_login_request_from_another_device">Bitwarden can notify you each time you receive a new login request from another device.</string>
   <string name="skip_for_now">Skip for now</string>
   <string name="done_text">Done</string>
+    <string name="page_number_x_of_y">%1$s of %2$s</string>
```

Comment:
Wanna remove the extra spaces here to match the rest.

Reply:
oof, ty!